src/app/extensions/quoting/store/quoting/quoting.effects.ts

```diff
@@ -122,10 +124,10 @@ export class QuotingEffects {
     this.actions$.pipe(
       ofType(deleteQuoteFromBasket),
       mapToPayloadProperty('id'),
-      withLatestFrom(this.store.pipe(select(getCurrentBasket))),
-      map(([quoteId, basket]) => basket?.lineItems?.find(li => li.quote === quoteId)?.id),
       whenTruthy(),
-      map(itemId => deleteBasketItem({ itemId }))
+      concatMap(quoteId => this.basketService.deleteQuoteFromBasket(quoteId)),
+      mergeMap(() => [loadBasket(), deleteQuoteFromBasketSuccess()]),
```

Comment:
Why is a BasketInfo object returned from the basket service, but not used for the following actions?

Reply:
A success message should be displayed